source/daplink/RTX_Config.c

```diff
@@ -90,7 +90,7 @@
 #elif defined(INTERFACE_SAM3U2C)
 #define OS_CLOCK    96000000
 #elif defined(INTERFACE_LPC4322)
-#define OS_CLOCK    204000000
+#define OS_CLOCK    96000000
```

Comment:
I am not sure when this was tested. My experience with the current DAPLink code was it was not stable at the higher frequency. Hence I have slowed down to the default power up frequency.  I am checking internally for information on how to safely raise the core frequency.
Once I receive this, I will try it and submit a separate PR to raise the core frequency.